cms/templates/visibility_editor.html

```diff
@@ -27,8 +27,17 @@ <h4 class="title">${_('You have not set up any groups to manage visibility with.
         </div>
     % elif is_staff_locked:
         <div class="summary-message summary-message-warning visibility-summary-message">
-            <i class="icon icon-warning-sign"></i>
-            <p class="copy">${_('The Unit this component is contained in is hidden from students. Visibility settings here will be trumped by this.')}</p>
+            <i class="icon fa fa-exclamation-triangle" aria-hidden="true"></i>
+            <p class="copy">
+                ## Translators: The "Warning:" text is meant to communicate the type of message being communicated and should always end with an extra space
```

Comment:
Now that the message is all in one string, this translator note probably isn't needed. But if you think it is still helpful, it is fine to leave it in.